apps/web/.eslintrc.js

```diff
@@ -35,13 +35,15 @@ module.exports = {
         format: ['PascalCase', 'camelCase', 'UPPER_CASE'],
       },
     ],
+    "react-hooks/rules-of-hooks": "error",
+    "react-hooks/exhaustive-deps": "warn"
```

Comment:
configuration as they suggest in the docs: https://www.npmjs.com/package/eslint-plugin-react-hooks